segmentstore/storage/src/test/java/io/pravega/segmentstore/storage/StorageTestBase.java

```diff
@@ -497,6 +499,14 @@ protected void createSegment(String name, Storage s) {
         s.create(name, null).join();
     }
 
+    protected String createSegmentName(String originName) {
+        if (isTestingSystemSegment) {
+            return "_" + originName;
```

Comment:
I'm pretty sure system segments are prefixed with `_system`. Are there other system segments that are not prefixed with that?

Reply:
Yes in production code when fully qualified segment names are used, it start from "_system"; however in test code, too many shortcut system segment names start with "_".
It seems not practical to start using all fully qualified segment name for all scenarios.